src/lists/vector.js

```diff
@@ -0,0 +1,74 @@
+const Node = require('../base')
+const iavector = require('iavector')
+const Block = require('@ipld/block')
+
+const _type = 'IPLD/Experimental/Vector/0'
+
+class Vector extends Node {
+  get _type () {
+    return _type
+  }
+
+  get (args) {
+    let path = args.path.split('/').filter(Boolean)
+    let index = parseInt(path.shift(), 10)
+
+    if (!Number.isInteger(index)) {
+      throw new Error('Vector type can only look up integers ' + index + ', ' + args)
```

Comment:
You could use template strings for the error message if you like.

Reply:
debugging leftover, I've reverted to what it was intended to be : `throw new Error('Vector type can only look up integers')`